### DISCUSSION TITLE
QT Serialbus devices not available in V204 windows build

### DISCUSSION THREAD

**Discussion by aidsevo** [collin80/SavvyCAN#399] [General]
As per my pic below, I just tried V204 latest dev build (in Windows) but the QT serial device connection option is greyed out and cant be selected. 
I tried V204 release (1.0.245), the QT device is not greyed out in that version but the "port"drop-down is empty.   Yes I have a QT serial pass-thru device plugged in when testing.  

V199 seems to be the last one that worked on Windows.

 <img src="https://i.imgur.com/in4KLRk.png"/>

**Comment by aidsevo** [comment]
I just tried the latest build today, I can now select QT serial Devices, but my device is missing from the port selection.  It works on V199. 
Is there any chance this could be solved?
Im using the windows build.

<img src="https://i.imgur.com/mPvIoDl.png"/>

<img src="https://i.imgur.com/cKxYIkd.png"/>

**Comment by collin80** [comment]
Weird. Yes, recently I did fix the problem where all the CAN plugins were not packaged. I don't know why the plugins would load now but then not find your connected hardware. I'll have to look into this.
